examples/cruise-control/cruise-control.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
```

Comment:
Since this is a `Kafka` resource, I would expect it to be in `examples/kafka/`. But looking below it's a copy of a config you already added to `examples/kafka/`.